requirements/base.txt

```diff
@@ -119,17 +120,16 @@ pbr==5.11.1
     # via stevedore
 polib==1.2.0
     # via edx-i18n-tools
-psutil==5.9.4
+psutil==5.9.5
     # via edx-django-utils
 pycparser==2.21
     # via cffi
 pymongo==3.13.0
     # via edx-opaque-keys
 pynacl==1.5.0
     # via edx-django-utils
-python-dateutil==2.4.0
+python-dateutil==2.8.2
```

Comment:
edx-platform is using 2.8.2
https://github.com/openedx/edx-platform/blob/master/requirements/edx/base.txt#L903